pkg/api/openapi/model_resource.go

```diff
@@ -406,6 +472,12 @@ func (o Resource) ToMap() (map[string]interface{}, error) {
 	if !IsNil(o.Manifest) {
 		toSerialize["manifest"] = o.Manifest
 	}
+	if !IsNil(o.DeleteOption) {
+		toSerialize["delete_option"] = o.DeleteOption
```

Comment:
delete_option or deleteOption?

Reply:
in restful api, it seems we should use `delete_option`. for example: `consumer_name` `create_at` etc https://github.com/openshift-online/maestro?tab=readme-ov-file#get-your-resource